src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -3266,7 +3266,7 @@ static void _svgLoaderParserXmlClose(SvgLoaderData* loader, const char* content)
     }
 
     for (unsigned int i = 0; i < sizeof(graphicsTags) / sizeof(graphicsTags[0]); i++) {
-        if (!strncmp(content, graphicsTags[i].tag, graphicsTags[i].sz - 1)) {
+        if (!strncmp(content, graphicsTags[i].tag, graphicsTags[i].sz - 1) && !isalpha(content[graphicsTags[i].sz - 1])) {
```

Comment:
isalpha is locale sensitive... and it's not clear for me what should happen if isalpha gets a space (I expected it to return false, but it returns true - I was checking the case `<line ...><\line>`).
anyway, imo the most elegant way would be to extract the tagname as it is done in the '_svgLoaderParserXmlOpen'.

Reply:
@JSUYA ofc I'll review as many times as needed.
I now the code grows and grows... since there are identical parts of code in open and close functions, would you consider to introduce a fun handling it? or do you think it's a bad idea? 
if it's a bad idea, I'll approve, since I've already checked and beside one small comment it's ok

Reply:
@mgrudzinska Thanks! Similar code already exists in _svgLoaderParserXmlOpen and _svgLoaderParserForValidCheckXmlOpen. I think it would be better to handle this as a new PR since it goes outside the purpose of this patch.  :)

Reply:
ok